app/controllers/api/v2/containers_controller.rb

```diff
@@ -44,7 +45,7 @@ def show
                                             Format should be repository:tag, e.g: centos:7')
           param :tty, :bool
           param :entrypoint, String
-          param :cmd, String
+          param :command, String, :required => true
```

Comment:
Docker seems to need a command. Also, the wizard code calls it `command` and not `cmd`.